src/main/java/org/example/StudRepo.java

```diff
@@ -0,0 +1,170 @@
+     * @return list of students
+     * @throws ValidationException
+     */
+    public List<Student> addStudent(String firstName, String lastName, LocalDate dateOfBirth, String genderString, String id) throws ValidationException {
+        if ((firstName == null) || (firstName.equalsIgnoreCase("")) || (lastName == null)
+                || (lastName.equalsIgnoreCase(""))) {
+            throw new ValidationException("First name and Last name should not be empty.");
+        }
+        if ((dateOfBirth.getYear() < 1900) || (dateOfBirth.getYear() > currentDate.getYear() - 18)) {
+            throw new ValidationException("Date of birth should be between 1900 and current year - 18.");
+        }
+        if (!((genderString.equalsIgnoreCase("FEMALE"))
+                || (genderString.equalsIgnoreCase("MALE"))
+                || (genderString.equalsIgnoreCase("F"))
+                || genderString.equalsIgnoreCase("M"))) {
+            throw new ValidationException("Gender should be male or female.");
+        }
+        if (id.length() != 13) {
+            throw new ValidationException("Id must have 13 characters.");
+        }
+        String testId = "279146358279";
+        int count = 0;
+        int lastNumofId = Integer.parseInt(String.valueOf(id.charAt(12)));
+        for (int i = 0; i < testId.length(); i++) {
+            count += Integer.parseInt(String.valueOf(testId.charAt(i))) * Integer.parseInt(String.valueOf(id.charAt(i)));
+        }
+        if (((count % 11 == 10) && (lastNumofId != 1)) || (((count % 11) < 10) && (lastNumofId != count % 11))) {
+            throw new ValidationException("Id is not valid");
+        }
+        if ((genderString.equalsIgnoreCase("FEMALE"))
+                || (genderString.equalsIgnoreCase("F"))) {
+            students.add(new Student(firstName, lastName, dateOfBirth, Gender.FEMALE, id));
+        } else {
+            students.add(new Student(firstName, lastName, dateOfBirth, Gender.MALE, id));
+        }
+        return students;
+    }
+
+    /**
+     * Deleting a student by ID (identifier)
+     *
+     * @param idToRemove should not be empty
+     * @return list of students
+     * @throws ValidationException
+     */
+    public List<Student> deleteStudent(String idToRemove) throws ValidationException {
+        if (idToRemove == null || idToRemove.equals("")) {
+            throw new ValidationException("ID is empty.");
+        }
+        boolean found = false;
+        for (Student student : students) {
+            if (((student.id).equals(idToRemove))) {
+                found = true;
+                students.remove(student);
+                System.out.println("Student with ID " + idToRemove + " was deleted");
+                break;
+            }
+        }
+        if (!found) {
+            throw new ValidationException("Student with ID " + idToRemove + " was not found.");
+        }
+        return students;
```

Comment:
return only the found variable to confirm you deleted or the id of the deleted student